ts/src/xt.ts

```diff
@@ -4601,10 +4601,11 @@ export default class xt extends Exchange {
         if (signed) {
             this.checkRequiredCredentials ();
             const defaultRecvWindow = this.safeString (this.options, 'recvWindow');
-            const recvWindow = this.safeString (params, 'recvWindow', defaultRecvWindow);
+            const recvWindow = this.safeString (query, 'recvWindow', defaultRecvWindow);
             const timestamp = this.numberToString (this.nonce ());
-            body = params;
+            body = query;
             if ((payload === '/v4/order') || (payload === '/future/trade/v1/order/create') || (payload === '/future/trade/v1/entrust/create-plan') || (payload === '/future/trade/v1/entrust/create-profit') || (payload === '/future/trade/v1/order/create-batch')) {
+                body = this.extend (body);
```

Comment:
Almost there, yet, no need to extend the body here, since `.omit` also does a copy, so after the `omit` call + `body = query` we can inject directly into the `body`.

Reply:
uh oh, right, in fast switching i make dull mistakes

Reply:
@ttodua ok, plz test this edit in xt by issuing a private call (fetchBalance or createOrder) just to make sure we're not breaking anything.

Reply:
fetchBalance worked well, for createOrder I have zero funds in XT account,will transfer & test in 10 mins

Reply:
though, still went and returned insufficientFunds

Reply:
createOrder works too, confirmed